### DISCUSSION TITLE
Urgent Appeal for Action: Addressing Abuse of Power on the US Team Talk Server

### DISCUSSION THREAD

**Discussion by muamel1** [BearWare/TeamTalk5#2241] [General]
@bear101 I am writing to address a concerning issue that has recently taken place on the US Team Talk server. It has come to my attention that an admin, named Shamika, known as Red Queen, has made unwarranted decisions and abused their power within the server.
It has become evident that Red Queen has been abusing her power on the server for personal reasons, resulting in unjust bans and creating an overall hostile environment. It appears that Red Queen has been using her authority to settle personal vendettas, rather than upholding the values of fairness and inclusivity.
Multiple instances have occurred where Red Queen has banned individuals solely because of personal conflicts or disagreements. It is deeply concerning to witness such misuse of power, as it not only disrupts the harmony of the community but also undermines the trust and integrity of the server as a whole.
Moreover, the recent banishment of several members, including myself, has shed light on the extent of Red Queen's arbitrary actions. It has come to our attention that one of us was wrongfully accused of insulting Red Queen, leading to our collective banishment from the server. This hasty decision reflects a lack of impartiality and discernment on Red Queen's part.
Additionally, Red Queen has displayed a blatant disregard for protocol and fairness by moving herself into our private locked channel and openly declaring her ability to abuse her power without consequence. This behavior further exacerbates the toxic environment within the server, leaving members feeling unsafe and unwelcome.
Furthermore, Red Queen's apparent confidence that she is immune to punishment and can act with impunity is alarming and only serves to embolden her unjust actions.
It is evident that Red Queen is allowing personal grievances to cloud her judgment, leading to the unjust punishment of innocent members. Such behavior not only tarnishes the reputation of the US Team Talk server but also creates a hostile atmosphere detrimental to the community's well-being.
I implore you to take swift and decisive action to address this issue and restore order to the US Team Talk server. It is crucial that measures are put in place to prevent further abuse of power and ensure that all members are treated fairly and respectfully.
Thank you for your attention to this matter. I trust that you will take the necessary steps to rectify the situation and uphold the values of our community.

**Comment by blind-masters** [comment]
I agree that her behavior includes using her admin powers for her own personal gain, as well as spreading rumors against anyone who does not agree with her opinion.

**Reply by muamel1** [reply]
believe me I was avoiding talking to her because I know she would cause a problem with me as soon as I disagree with her even in a simple opinion.
I remember when she banned my friend Ijlal from the server just because he said that NVDA is better than JAWS, so I was avoiding talking to her. Despite that, she banned me and all my friends just because she claims that one of them offended her.

**Comment by shadi-abu-affash** [comment]
I agree, especially recently, her behavior has become increasingly hostile towards those she dislikes.
Let me share a couple of incidents that illustrate this point. Once, while I was in the US server, specifically in the "Music Room US" channel, I made a joke that wasn't appropriate. In response, she banned me from the entire server.
On another occasion, I mistakenly spoke in Arabic in a channel designated for English-only communication, and she promptly banned me from the entire server.
The main issue here is that she seems to prioritize her personal grudges over her responsibilities as an admin.
Lastly, whenever someone threatens to report her, she dismisses it by saying that the developer of TeamTalk won't care, effectively exploiting this situation to her advantage.

**Comment by karon-sarbest** [comment]
I completely agree on everything that it's mentioned in  this subject, Regarding Red Queen abusing her power on the US teamtalk server, Once I was playing some game called  QuentinC's Gameroom, And when I was online she messaged me, and  Because of some jokes that I did with her on that game  i was jjokeing with her for like an minute, And after that when I logged into the US Server I got an error it says your band from the server, thank you really for talking about this problem

**Comment by muamel1** [comment]
It's disappointing that instead of using her moderator powers to stop spam, she is using them to ban people she has personal problems with. What's even more frustrating is that she doesn't seem to know how to unban the people she has banned. This has happened multiple times, where she has banned someone and then later apologized and said she doesn't know how to unban them.
I don't understand why someone who is so impulsive and lacks the necessary computer skills would be given moderator powers on the official server. She is clearly not fit for the role, and her actions are only making things worse.

**Reply by karon-sarbest** [reply]
ones she banned me, and after that i told her to unbann me she said ok, i even sent her my IP, she did not know 
how to unban me again

**Reply by shadi-abu-affash** [reply]
I have the same story.
Once she band me from her channel and when i joined with a vpn i asked her to un band me she said i don't know

**Comment by computertech1012** [comment]
Hi,
I agree with what everyone is saying in this discussion. Shamika is an abusive administrator and tells lies to cover it up. Clames that the dev of TeamTalk and her talk on messenger and she tells him things that are not true about several people. Also creates files and uploads them as herself and or makes other bearware usernames to cover it up. If she's a moderator, she should not do those things about anyone, she should remain neutral and not make problems with people and ban users for her own personal reasons.
Thanks for reading!

**Reply by computertech1012** [reply]
Hi,
I can also remember one time when some users were in a private channel, Shamika moved herself into their channel along with another administrator named Saeed. He is Mohammed and Shamika told lies to Bjorn in order to get Saeed reinstated as an administrator. He is also yet another abusive administrator on the official US server.
Thanks for reading!

**Reply by muamel1** [reply]
Exactly, she always says that Bjorn listens to her no matter what she does.

**Reply by bear101** [reply]
@Muamalaljanahi you have a long record spamming and causing disturbance, so I choose to trust Shamika instead of you. Please shut up and find another server to join. Your presence on the US server is clearly not wanted.

**Reply by muamel1** [reply]
@bear101 Wow man, it seems like she has filled your mind with lies for you to respond in this way, which is the least I can say disrespectful. The only person everyone agrees has a long record of spamming and causing disturbance is @Ramon-Salazar , Shamika's friend: he is banned from most other platforms.
Don't worry about me, I have my own server where we are safe from the reckless actions of the Stupid admin on your server.
Have a good day and keep trusting everything Shamika says

**Reply by computertech1012** [reply]
Hi,
Not only is it disrespectful, but it is also unprofessional for an owner of a company to speak to people in this way. Especially in a public setting.
Thanks for reading!

**Reply by muamel1** [reply]
He accuses me of being a spammer, and his evidence is the word of someone who hates me and is willing to fabricate accusations against me. Believe me, if I had known that he lacks the decency to talk to people and accuses users without evidence, I would not have wasted my time writing this discussion.

**Comment by jimmytruth** [comment]
I strongly disagree.

Regarding the auther of the topic: (Muamalaljanahi), and his friends including blindmasters have been banned for too many reasons, recently because of spreading misinformation, racism, spamming, harassing and making nicknames about people.

As I know, and since I used to be in the server with my friends a lot Shamika never banned people for no reason, and also she never moved herself  to the privat (password protected) channels, and also she isn't racist.

She's taking actions against those people who spread misinformation seriously, and those who got banned for no reason probably they got banned by Spambot, either their IP got reported or they did too many actions at once, like joining and leaving channels many times.

**Reply by Ramon-Salazar** [reply]
Agree, Muamalaljanahi is in fact racist, sometimes against turks, did too many things in the past including making abusive nicknames about the people including my friends until now, talking about them behind their backs in public channels, used to make scripts to spam The official teamtalk 5 server (US), especially when he didn't get his views accepted.

I won't be surprised his friends supporting him, since most of them are also banned for same reasons especially one of them knew the truth about them so they made sexual and offensive nicknames about him and his family.

They aren't  banned for personal reasons as they claim

**Reply by muamel1** [reply]
@jimmytruth or DR. Ramon Salazar While you were spamming the server, I was developing [SpamKiller](https://github.com/Muamalaljanahi/SpamKiller) the first bot that was able to stop you from spamming: At the time, I gave it to Shamika after it became clear that she was incapable of carrying out her duties as a server administrator.
about 10 of us were banned from the server 3 days ago because someone made a name about her and it turned out to be someone from outside our group. He admitted to it, but surprisingly he is still on the server and she did nothing to him. Therefore, it is highly likely that this is a conspiracy to ban us from the server because of our personal problems with her.
Regarding moving herself to password-protected channels: I swear to God, once we had a problem with her boyfriend and we went to a private channel to avoid talking to him. After about 20 minutes, she moved herself and the person concerned to our password-protected channel without warning. The server has logging enabled, so @bear101 can check the logs himself.
Everything I say has witnesses and evidence, and I don't need to lie to prove my case.

**Reply by jimmytruth** [reply]
That isn't true, you were mad because when someone used the script against your server you developed your own spam killer, but that even isn't perfect and can't ban too many IPS regarded as proxies.

It appeared that You're mad because it turned out that Ramon became against you and he's no longer your friend, after he realized how close-minded you are, though there were used to be records that you are trying the best of helping him spam the US Server 24/7 long time ago and later you failed miserably.

Nowadays everyone know how close-minded, racist and spammer you are , you spam everyone who don't agree with your views, same goes for your friends, so won't be surprised why Shamika banned your friends.

> about 10 of us were banned from the server 3 days ago because someone made a name about her and it turned out to be someone from outside our group. He admitted to it, but surprisingly he is still on the server and she did nothing to him.

LOL I'm wondering who are the other 7 people? as I know your other 3 friends got banned for making abusive nicknames about her and also for hateful speech.

It turned out you are against someone used to be your friend just because he used to be in your group, when he blocked all of you you and your friends started making these nicknames, so stop accusing someone who didn't enter to the server of making these nicknames, we all know the truth even @bear101

**Reply by muamel1** [reply]
@Ramon-Salazar If even 30% of what you said was true, I would have responded to you. But as usual, you are personalizing the issue as if I am the only one who was unfairly banned, in order to cover up her mistakes. If it's not me, then there are many others who were banned by Shamika just because she doesn't like them: for example @karon-sarbest and @computertech1012, and many others who I spoke to but they don't use GitHub.
Enjoy spreading rumors about me. Everyone who knows me knows that what you say is just lies.

**Reply by Ramon-Salazar** [reply]
@Muamalaljanahi What @jimmytruth  said is 100%true, nothing found to be wrong.

@karon-sarbest  got banned probably by spambot, already mentioned earlier and you already noticed that, probably he got banned before for abusive behaviours, as I know him well he randomly insulting people while he's being so angry as a sudden.

Regarding Cookie AKA @computertech1012 and @johnrechard85  he's a known spammer and drama causer, won't be surprised he's beeing fairly banned by all of the 3 Moderators.

You aren't the only one who I mentioned, but all of your friends including those who aren't in github and you know well who I meant.

All of your 3 friends including Justen and @shadi-abu-affash are cleary unwanted on the server and the moderators are doing the best to make it free from Drama and unwanted bahaviours that you and your friends are causing.

@bear101  is 100%right you better find a server, since you already have your own server  and you claim to be a safe server why are you being so upset about that you and your friens  being banned from The Official Server if you can simply move your friends there?

**Reply by johnrechard85** [reply]
The issue lies in the fact that this server holds an official status, and numerous individuals intend to socialize and enjoy themselves, rather than facing arbitrary bans. As I mentioned earlier, I'm not defending anyone; it's essential to grasp this before accusing anyone.

**Comment by johnrechard85** [comment]
Hi, I expressed similar thoughts last year, but unfortunately, my concerns went unnoticed. It is important to address the fact that Muannal is also involved in spamming activities. However, Shamica's behavior has crossed a line, and actions need to be taken. Despite this, Bare101 seems to be blinded by Shamica's charm and only trusts her. Shamica continuously uploads numerous files, showing a lack of respect towards everyone. Regrettably, Bare101's attachment to Shamica has led to neglecting important matters concerning the official servers. Thank you for taking the time to read this.

**Comment by johnrechard85** [comment]
Allow me to provide further clarification. Shamica frequently bans individuals arbitrarily and consistently uploads files indiscriminately about various people. There seems to be no oversight to manage this behavior, making the server unsafe for users. As a suggestion, I advise Bare101 to take action against Shamica and appoint a more suitable individual if he values the reputation of his servers.
